packages/admin-api/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@tryghost/admin-api",
+  "version": "0.0.0",
+  "repository": "https://github.com/TryGhost/Ghost-SDKs/tree/master/packages/admin-api",
+  "author": "Ghost Foundation",
+  "license": "MIT",
+  "main": "index.js",
+  "scripts": {
+    "dev": "echo \"Implement me!\"",
+    "test": "NODE_ENV=testing mocha ./test/**/*.test.js ./test/*.test.js",
+    "lint": "eslint . --ext .js --cache",
+    "posttest": "yarn lint"
+  },
```

Comment:
Found another slimer bug (again fixed it and updated) this block should be inserted here:

```
 "publishConfig": {
    "access": "public"
  },
```